compile.sh

```diff
@@ -27,3 +27,4 @@ swiftc -O -parse-as-library -Xcc -funroll-loops -Xcc -march=native -Xcc -ftree-v
 #dotnet publish csharp/csharp.csproj -o csharp/code-aot /p:PublishAot=true
 dotnet publish csharp/csharp.csproj -o csharp/code
 ghc -O2 -fllvm haskell/code.hs -o haskell/code || { echo "ghc: cannot compile with llvm backend; fallback to use default backend"; ghc -O2 haskell/code.hs -o haskell/code; }
+cd clojure && mkdir -p classes && clojure -Sdeps '{:paths ["."]}' -M -e "(compile 'code)"
```

Comment:
I think it's a good idea to restore the working directory back after we `cd`, so that other languages added after this don't need to track the current directory changes 🙏

Reply:
Good catch. I actually ran in to this with the `inko` compile, since I don't have it, it crashed out before it could `cd` back up again and the rest of the compiles failed. I got around that by executing the `inko` commands in a sub shell. Added that to the Clojure compile now. WDYT?